wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests/Angular2CalculatorTest.wiki

```diff
@@ -0,0 +1,28 @@
+Sample test on [[Angular][https://angular.io/]] application. 
+
+!note '''Please note:''' The application used in this test does NOT seem to work with !-PhantomJs-!. It does work when executed with Chrome or Firefox.
+
+!note Due to the nature of [[Plunker][https://plnkr.co/]] which hosts the application in an iframe, the page must first be opened with !-BrowserTest-!, switch to the embedded iframe and wait for the application to load.
```

Comment:
The only problem I found with both the sites I tried was that they didn't need Ng2BrowserTest, the sample tests work just as well with BrowserTest ;-)
(This is actually similar to what I found for AngularJs sites: I tend not to need NgBrowserTest and just use BrowserTest for my tests.)

Is there any particular feature you found in your own apps that Ng2BrowserTest was needed for? It would be good if we could find a sample that really relied on the specialised fixture...